web/ugahacks-8/pages/components/Header/Header.tsx

```diff
@@ -88,7 +88,7 @@ export default function Header() {
                 Pre-Register
               </a>
             </div>
-            <div className="flex ml-auto pr-4">
+            <div className="flex ml-auto pr-4 [visibility:hidden] md:visible">
```

Comment:
Had to do `[visibility:hidden]` instead of `invisible` due to bootstrap.css making their invisible class important.

Reply:
Are you using bootstrap for this?

Reply:
Nvm I see what you are doing now, looks good